VideoLocker/AndroidManifest.xml

```diff
@@ -93,28 +93,27 @@
 
         <activity
             android:name=".view.CourseDiscussionTopicsActivity"
+            android:configChanges="orientation|keyboardHidden|screenSize"
```

Comment:
Agreed.

For future reference, here's what the official docs say:
https://developer.android.com/guide/topics/resources/runtime-changes.html#HandlingTheChange
> Note: Handling the configuration change yourself can make it much more difficult to use alternative resources, because the system does not automatically apply them for you. This technique should be considered a last resort when you must avoid restarts due to a configuration change and is not recommended for most applications.

And another explanatory article on config-changes best practices:
http://www.androiddesignpatterns.com/2013/04/retaining-objects-across-config-changes.html